components/SMSSendForm.tsx

```diff
@@ -0,0 +1,20 @@
+type SMSProps = React.PropsWithChildren<{
+  memberID: string,
+  orgID: string,
+}>;
+export const SMSSendForm = ({ memberID, orgID }: SMSProps) => {
+  return (
```

Comment:
Could we add some copy along the lines of "We need to enroll your phone # for MFA"?

Reply:
https://github.com/stytchauth/stytch-b2b-nextjs-example/pull/20/commits/9ef1a7368fdd456d0c6c5078da564782be60e7ac